lib/node_modules/@stdlib/math/base/special/cinvf/test/test.js

```diff
@@ -0,0 +1,446 @@
+* limitations under the License.
+*/
+
+/* eslint-disable id-length */
+
+'use strict';
+
+// MODULES //
+
+var tape = require( 'tape' );
+var isnanf = require( '@stdlib/math/base/assert/is-nanf' );
+var absf = require( '@stdlib/math/base/special/absf' );
+var EPS = require( '@stdlib/constants/float32/eps' );
+var FLOAT32_SMALLEST_NORMAL = require( '@stdlib/constants/float32/smallest-normal' );
+var Complex64 = require( '@stdlib/complex/float32/ctor' );
+var real = require( '@stdlib/complex/float32/real' );
+var imag = require( '@stdlib/complex/float32/imag' );
+var cinvf = require( './../lib' );
+
+
+// FIXTURES //
+
+var data = require( './fixtures/julia/data.json' );
+var largeNegativeImaginaryComponents = require( './fixtures/julia/large_negative_imaginary_components.json' );
+var largeNegativeRealComponents = require( './fixtures/julia/large_negative_real_components.json' );
+var largePositiveImaginaryComponents = require( './fixtures/julia/large_positive_imaginary_components.json' );
+var largePositiveRealComponents = require( './fixtures/julia/large_positive_real_components.json' );
+var tinyNegativeImaginaryComponents = require( './fixtures/julia/tiny_negative_imaginary_components.json' );
+var tinyNegativeRealComponents = require( './fixtures/julia/tiny_negative_real_components.json' );
+var tinyPositiveImaginaryComponents = require( './fixtures/julia/tiny_positive_imaginary_components.json' );
+var tinyPositiveRealComponents = require( './fixtures/julia/tiny_positive_real_components.json' );
+
+
+// TESTS //
+
+tape( 'main export is a function', function test( t ) {
+	t.ok( true, __filename );
+	t.strictEqual( typeof cinvf, 'function', 'main export is a function' );
+	t.end();
+});
+
+tape( 'the function computes the inverse of a single-precision complex floating-point number', function test( t ) {
+	var v;
+
+	v = cinvf( new Complex64( 2.0, 4.0 ) );
+	t.strictEqual( real( v ), 0.10000000149011612, 'returns expected value' );
+	t.strictEqual( imag( v ), -0.20000000298023224, 'returns expected value' );
+	t.end();
+});
+
+tape( 'the function computes a complex inverse', function test( t ) {
+	var delta;
+	var qre;
+	var qim;
+	var tol;
+	var re;
+	var im;
+	var i;
+	var q;
+
+	re = data.re;
+	im = data.im;
+	qre = data.qre;
+	qim = data.qim;
+
+	for ( i = 0; i < re.length; i++ ) {
+		q = cinvf( new Complex64( re[ i ], im[ i ] ) );
+
+		if ( real( q ) === qre[ i ] ) {
+			t.strictEqual( real( q ), qre[ i ], 'returns expected real component' );
+		} else {
+			delta = absf( real( q ) - qre[ i ] );
+			tol = 3 * EPS * absf( qre[ i ] );
+			t.ok( delta <= tol, 'within tolerance. x: '+re[i]+'+ '+im[i]+'i. real: '+real( q )+'. expected: '+qre[i]+'. delta: '+delta+'. tol: '+tol+'.' );
+		}
+		if ( imag( q ) === qim[ i ] ) {
+			t.strictEqual( imag( q ), qim[ i ], 'returns expected imaginary component' );
+		} else {
+			delta = absf( imag( q ) - qim[ i ] );
+			tol = 3 * EPS * absf( qim[ i ] );
```

Comment:
```suggestion
			tol = 3.0 * EPS * absf( qim[ i ] );
```

Reply:
@aayush0325 This decimal representation should be followed here, below and in `test.native.js`.

Reply:
I've updated this now. thanks!